includes/managers/widgets.php

```diff
@@ -255,7 +269,7 @@ public function register( Widget_Base $widget_instance ) {
 		 * @since 3.18.0
 		 *
 		 * @param bool $should_register Should widget be registered. Default is `true`.
-		 * @param \Elementor\Widget_Base $widget_instance Widget instance.
+		 * @param \Elementor\$widget_instance Widget instance.
```

Comment:
We can return the type here

Reply:
totally right, missed that in my cleanup by accident